net/web/src/constants/Strings.js

```diff
@@ -1252,3 +1252,212 @@ export const ru = {
   confirmDisable: 'Отключение двухфакторной аутентификации',
   disablePrompt: 'Вы уверены, что хотите отключить двухфакторную аутентификацию?',
 };
+
+export const el = {
+  code: 'el',
+  settings: 'Ρυθμίσεις',
+  contacts: 'Επαφές',
+  logout: 'Αποσύνδεση',
+  confirmLogout: 'Είστε σίγουροι ότι θέλετε να αποσυνδεθείτε;',
+  contactsUpdated: 'Η κατάσταση των επαφών ενημερώθηκε',
+  disconnected: 'Αποσυνδεθήκατε από το διακομιστή',
+  allDevices: 'Αποσύνδεση από όλες τις συσκευές',
+  ok: 'Εντάξει',
+  cancel: 'Ακύρωση',
+  enableNotifications: 'Ειδοποιήσεις',
+
+  new: 'Νέο',
+  newMessage: 'Νέο Μήνυμα',
+  topics: 'Συζητήσεις',
+  unsetSealing: 'Κατάργηση Κλειδιού Σφράγισης',
+  newTopic: 'Νέα Συζήτηση',
+
+  noContacts: 'Δεν υπάρχουν Επαφές',
+  noTopics: 'Δεν υπάρχουν Συζητήσεις',
+  noConnected: 'Δεν υπάρχουν Συνδεδεμένες Επαφές',
+  subjectOptional: 'Θέμα (προαιρετικό)',
+  members: 'Μέλη',
+  sealedTopic: 'Σφραγισμένο Θέμα',
+  start: 'Έναρξη',
+
+  communication: 'Επικοινωνία για τον Αποκεντρωμένο Ιστό',
+  setupProfile: 'Ρύθμιση του προφίλ σας',
```

Comment:
Better to do it this way 'Ρυθμίστε το προφίλ σας' and NOT 'Ρύθμιση του προφίλ σας' that i have done it because it is directly addressing the user and to maintain consistency with the translation style of 'Start a conversation'